pkg/rest/client.go

```diff
@@ -186,7 +187,7 @@ func (c Client) executeRequest(request *http.Request) (Response, error) {
 		}
 
 		if c.trafficLogger != nil {
-			err := c.trafficLogger.Log(request, reqBody, resp, string(respBody))
+			err := c.trafficLogger.Log(request, io.NopCloser(strings.NewReader(reqBody)), resp, io.NopCloser(bytes.NewReader(respBody)))
```

Comment:
doesn't matter much, but couldn't you leave the interface of Log unchanged and just do this wrapping into a ReadCloser inside of that before calling the internal log methods?

Reply:
sure, I've thought about it but as you've already stated, it doesn't really matter :D 
But I will do it, yeah.